src/coreComponents/fieldSpecification/FieldSpecificationBase.cpp

```diff
@@ -110,6 +110,10 @@ void FieldSpecificationBase::setMeshObjectPath( Group const & meshBodies )
   }
   catch( std::exception const & e )
   {
+    g_errorLogger.currentErrorMsg()
+      .addToMsg( getWrapperDataContext( viewKeyStruct::objectPathString() ).toString() +
+                      " is a wrong objectPath: " + m_objectPath + "\n" )
+      .addContextInfo( getWrapperDataContext( viewKeyStruct::objectPathString() ).getContextInfo().setPriority( 2 ) );
```

Comment:
long line
you can either declare a variable for the `getWrapperDataContext()` or format it that way:
```suggestion
      .addContextInfo( getWrapperDataContext( viewKeyStruct::objectPathString() ).getContextInfo()
                         .setPriority( 2 ) );
```
... here and at the other `setPriority()` calls.